app/controllers/auth_controller.rb

```diff
@@ -31,6 +31,15 @@ def fetch_return_to_path
   def do_user_auth(auth)
     sign_out
     user = User.from_oauth_hash(auth)
+    unless ENV["NO_WELCOME_EMAIL"]
```

Comment:
ahhh ok makes sense - might be good to fix all those tests at some point rather than not sending the emails by default (the tests should probably be re-written to not actually git the auth controller for login purposes) but this seems like a good solution for now